internal/db/storage/blob/s3/s3.go

```diff
@@ -31,6 +31,11 @@ import (
 	"github.com/vdaas/vald/internal/errors"
 )
 
+var (
+	readerNew = reader.New
```

Comment:
I am not sure but do you think `newReaderFunc` is better?

Reply:
I think it lood to me.
thank you